src/applications/appeals/995/validations/date.js

```diff
@@ -3,7 +3,9 @@ import moment from 'moment';
 import { parseISODate } from 'platform/forms-system/src/js/helpers';
 
 import { fixDateFormat } from '../../shared/utils/replace';
-import { errorMessages, FORMAT_YMD, MAX_YEARS_PAST } from '../constants';
+import { errorMessages, FORMAT_YMD } from '../constants';
+
+import { MAX_YEARS_PAST } from '../../shared/constants';
```

Comment:
`FORMAT_YMD` is in shared constants

Reply:
Removed from constants files and updated across 995 & 996